Code/GraphMol/MolDraw2D/MolDraw2D.h

```diff
@@ -29,14 +29,39 @@
 #include <GraphMol/RDKitBase.h>
 #include <GraphMol/ChemReactions/Reaction.h>
 
-#include <boost/tuple/tuple.hpp>
-
 // ****************************************************************************
 using RDGeom::Point2D;
 
 namespace RDKit {
 
-typedef boost::tuple<float, float, float> DrawColour;
+struct DrawColour {
+  double r = 0.0, g = 0.0, b = 0.0, a = 1.0;
+  DrawColour(){};
+  DrawColour(double r, double g, double b, double a = 1.0)
+      : r(r), g(g), b(b), a(a){};
+  bool operator==(const DrawColour &other) const {
+    return r == other.r && g == other.g && b == other.b && a == other.a;
+  }
+  bool feq(const DrawColour &other, double tol = 0.001,
+           bool ignoreAlpha = true) const {
+    return fabs(r - other.r) <= tol && fabs(g - other.g) <= tol &&
+           fabs(b - other.b) <= tol &&
+           (ignoreAlpha || fabs(a - other.a) <= tol);
+  };
+  DrawColour operator+(const DrawColour &other) const {
```

Comment:
I'm a bit bothered by this as r,g,b,a could end up > 1.

Reply:
Could very well happen when doing average colors (i.e. `(c1+c2)/2`).
I think this kind of problem is best handled when the color is actually used